### PR TITLE
Refactor the WineShortcut Class

### PR DESCRIPTION
**Pull request by madoar**
This PR refactors the `WineShortcut` class with the "new" class syntax and some other newer language features.

### CODE REVIEW

File: Engines/Wine/Shortcuts/Wine/script.js [hunk 1]
```diff
@@ -1,186 +1,197 @@
...
-    return this;
-}
+    /**
+     * Sets the miniature for the shortcut
+     *
+     * @param {string[]|URI} miniature An array which specifies the application of which the miniature shall be used or URI of the miniature
+     * @returns {WineShortcut} The WineShortcut object
+     */
+    miniature(miniature) {
+        if (isArray(miniature)) {
```
Comment (madoar) [inline Engines/Wine/Shortcuts/Wine/script.js]: @plata @ImperatorS79 @qparis does someone of you know where the `isArray` function is defined? It looks strange to me that we can use a function which we neither defined nor imported...

Comment (plata) [reply Engines/Wine/Shortcuts/Wine/script.js]: https://docs.oracle.com/javase/8/docs/jdk/api/nashorn/jdk/nashorn/api/scripting/JSObject.html#isArray--

Comment (madoar) [reply Engines/Wine/Shortcuts/Wine/script.js]: Maybe we should try https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray?

Comment (plata) [reply Engines/Wine/Shortcuts/Wine/script.js]: Yes.

**Pull request merged**